packages/client/src/index.test.ts

```diff
@@ -45,3 +45,80 @@ describe('init client', () => {
     expect(codeVerifier.length).toBeGreaterThan(10);
   });
 });
+
+const access_token = 'TvuiR2kU_38PtQcZJM8NFF8I4-0lWnr8C5eWsTmCOON';
+const refresh_token = 'icveztdOZ879up1vSsKMFjLzscKBJ44kIoXMKlWeunN';
```

Comment:
we can just randomly generate this instead of hard-code if it doesn't matter for the test cases.